recipe/meta.yaml

```diff
@@ -2,6 +2,10 @@
 {% set build_num = 5 %}
```

Comment:
Let's change this to `20` and make a comment that it should not be reset to 0

Reply:
It's not by a big margin, but I don't think that's a better solution, for two reasons:
* new lapack versions _should_ IMO reset the build number (aside from being confusing for CF-regulars, the GH PR template invites to), and there's really two different constructs being versioned here (lapack & CF-blas-arch), which don't _have to_ have the same `build_num`
* in case there's ever a need to create a branch for rebuilding for an older lapack-version, it would be very intransparent what the right `build_num` should be

Reply:
I don't really agree with 1., but 2 is good enough. Can you add a comment above that if it is reset, then blas_minor offset needs to be updated.

Reply:
Thanks, done.

Reply:
As a matter of fact, what do you think about creating a 3.8 branch here and we do one more build that has a consistent build number & corresponding blas-version? Because currently, there's a bit of an eclectic [mix](https://anaconda.org/conda-forge/blas/files) with blas builds for 3.9 but with low numbers

Reply:
what does that achieve? builds with highest build number are 3.8 right now.

Reply:
Some of the low numbers (like `osx-arm64/blas-2.5-openblas.tar.bz2`) actually are for 3.9, and it's basically build-dependent whether it's 3.8 or 3.9 (for example a new `win-64/blas-2.2-openblas.tar.bz2` was built 2 months ago - I only know that it cannot be 3.9 because #55 hadn't been merged yet, but then why does it have such a low build_num?)

Reply:
So the idea would be to have a definite max build_num across 3.8 builds on all platforms, so that the resolver doesn't have to chase down rabbit holes based on the apparent patchwork of conditions (that lead to lower build_nums being more current).

Reply:
But what would creating a 3.8 branch achieve?

Reply:
max build_num across 3.8 builds is 21 on all platforms. what am I missing?

Reply:
With such a branch, we could have another 3.8 build `libblas-3.8.0-22_{blis,mkl,openblas}` that _uniformly_ corresponds to a `blas-2.22_{blis,mkl,openblas}`, without having to switch back and forth for the needs of 3.9/master.

Reply:
> max build_num across 3.8 builds is 21 on all platforms. what am I missing?

That the low `blas` build_nums (now `blas_minor`) got overwritten once lapack moved to 3.9.0, and now there's a mix of `blas` builds that either belong to 3.8/3.9, depending on whether the old package had been deleted/retired/etc.

e.g. `win-64/blas-2.2-openblas.tar.bz2` should be ancient, but it is one of the most recent [builds](https://anaconda.org/conda-forge/blas/files)

Reply:
Ah, I see what you are saying, but I'd like to avoid unnecessary builds. If you want, please send a PR to mark the 3.9.0 builds with low numbers as broken.

Reply:
OK cool, glad we're on the same page.

However, compared to identifying which packages exactly were wrongly overwritten, marking them broken (breaking the reproducibility of people's environments, and possible breaking lapack-3.8-only deployments completely), creating a branch and running CI for 2h seems like the much better trade-off?!

Reply:
I'm sorry, but I still don't see any advantage for a rebuild. Right now the ones with the highest build numbers are 3.8 and doing a rebuild wouldn't change this. (The only thing that would change is that they'll have the same build number.)

Reply:
Yes, the ones with the highest number are 3.8, but the highest number is often _not the most up-to-date_ 3.8 build (leading to outdated requirements e.g. with compilers etc.)! Again the example `win-64/blas-2.2-openblas.tar.bz2` (is definitely a 3.8 build, as it came before #55), which is the second-most-recent windows+openblas build...

Just have a look at the `blas`-[files](https://anaconda.org/conda-forge/blas/files) and see the "chaos" for yourself.

Reply:
Ah wait, sorry, now I confused myself with 3.9 vs MKL builds...

Reply:
Nope, `blas-2.2` is a 3.9 build.

Reply:
I realised... see updated comment above.

Reply:
It'll still be hard to follow as well. `netlib` builds are done with a different build number as well and I don't think we'll be able to keep the build numbers in sync for different blas implementations and IMO that's okay.

Reply:
Sure, but the `netlib` ones don't have an associated `blas` package.

Reply:
Nevermind, they do. OK, I'll drop this, thanks for the exchange

Reply:
Thanks for the discussion. I really appreciate your work on better BLAS support on conda-forge.

Reply:
Happy to hear it! :)